src/cli.rs

```diff
@@ -0,0 +1,19 @@
+use clap::Parser;
+use std::path::PathBuf;
+
+#[derive(Parser, Debug)]
+#[clap(version, about, long_about = None)]
+pub struct CliArgs {
+    #[clap(short, long)]
```

Comment:
add the default of clap.
There's already a helper function to help you in
https://github.com/storopoli/dead-man-switch/blob/8f5197ed082fab173a122d72db2d0bc39319f0cc/src/config.rs#L108